lib/ui/painting/multi_frame_codec.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_LIB_UI_PAINTING_MUTLI_FRAME_CODEC_H_
+#define FLUTTER_LIB_UI_PAINTING_MUTLI_FRAME_CODEC_H_
+
+#include "flutter/fml/macros.h"
+#include "flutter/lib/ui/painting/codec.h"
+
+namespace flutter {
+
+class MultiFrameCodec : public Codec {
+ public:
+  MultiFrameCodec(std::unique_ptr<SkCodec> codec);
+
+  ~MultiFrameCodec() override;
+
+  // |Codec|
+  int frameCount() const override;
+
+  // |Codec|
+  int repetitionCount() const override;
+
+  // |Codec|
+  Dart_Handle getNextFrame(Dart_Handle args) override;
+
+ private:
+  const std::unique_ptr<SkCodec> codec_;
+  const int frameCount_;
```

Comment:
Shouldn't these ivars be underscored, not camel case?

Reply:
This class was moved as-is from codec.cc to its own file. The dubious naming scheme is present because this component was originally in blink. None of the original code remains but the still stuck around. I'll change the this file and other to match engine style in a subsequent patch.